test/fixtures/tls-connect.js

```diff
@@ -6,18 +6,18 @@
 const common = require('../common');
 const fs = require('fs');
 const join = require('path').join;
+// Check if Node was compiled --without-ssl and if so exit early
+// as the require of tls will otherwise throw an Error.
+if (!common.hasCrypto) {
+  common.skip('missing crypto');
+  process.exit(0);
+}
 const tls = require('tls');
 const util = require('util');
 
-module.exports = exports = checkCrypto;
-
-function checkCrypto() {
-  if (!common.hasCrypto) {
-    common.skip('missing crypto');
-    process.exit(0);
-  }
+module.exports = exports = function() {
   return exports;
-}
+};
```

Comment:
You can remove this if you update the files that include this file:
```
$ git grep -n tls-connect test/
test/parallel/test-tls-addca.js:11:} = require(join(common.fixturesDir, 'tls-connect'))();
test/parallel/test-tls-ca-concat.js:10:} = require(join(common.fixturesDir, 'tls-connect'))();
test/parallel/test-tls-cert-chains-concat.js:10:} = require(join(common.fixturesDir, 'tls-connect'))();
test/parallel/test-tls-cert-chains-in-ca.js:10:} = require(join(common.fixturesDir, 'tls-connect'))();
test/parallel/test-tls-connect-secure-context.js:9:} = require(join(common.fixturesDir, 'tls-connect'))();
test/parallel/test-tls-peer-certificate.js:9:} = require(join(common.fixturesDir, 'tls-connect'))();
test/parallel/test-tls-socket-default-options.js:10:} = require(join(common.fixturesDir, 'tls-connect'));
```
Simply remove the `()` at the end.